packages/sui-test/bin/karma/index.js

```diff
@@ -39,10 +39,12 @@ module.exports = async ({
 
   configure.files = [
     `${CWD}/node_modules/@babel/polyfill/dist/polyfill.min.js`,
+    srcPattern && `${CWD}/${srcPattern}`,
```

Comment:
Following code added in L47 https://github.com/SUI-Components/sui/pull/834/files#diff-3259a4ee8eb62198bf98bcc989d16990R47

You can avoid filter undefined by doing this:
```suggestion
    ...(srcPattern && [`${CWD}/${srcPattern}`]),
```